builder/build/build.go

```diff
@@ -137,11 +137,17 @@ func (b *Build) buildWithDocker(config *containers.BuildConfig) error {
 
 func (b *Build) buildWithPack(config *containers.BuildConfig) error {
 	b.Log().Debug().Msg("pack config registry-mirrors")
-	cmd := exec.Command("pack", "config", "registry-mirrors", "add", "index.docker.io", "--mirror", DockerHubMirror)
+	builder := b.BuildpackBuilders()[0]
+	packBinary := "pack"
+	if builder == "heroku/buildpacks:20" {
```

Comment:
@ipmb Hi! Do you have metrics as to what proportion of your users are using the legacy `heroku/buildpacks:*` images (vs the `heroku/builder:*` images)? We had been planning on making the `heroku/buildpacks` repo private at some point to prevent people unknowingly using an insecure image (it hasn't had security updates since May 2024), but wouldn't want that to cause too much disruption to you/your users.

Reply:
Hey @edmorley! This is the blocker for moving our last user over to CNB https://github.com/heroku/buildpacks-deb-packages/issues/110